FlorenceBML/BMLacq306/BMLacq306.xml

```diff
@@ -223,8 +223,48 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <msPart xml:id="p1"><msIdentifier/><msContents><summary/>
                         <msItem xml:id="p1_i1">
                             <locus from="1ra" to="56rb"/> 
-                            <title type="incomplete" ref="LIT0000">Dəggwā</title>
+                            <title type="incomplete" ref="LIT3178Deggwa">Dəggwā for the whole year in an abridged version</title>
+                            <note>Several folia are lost. Does not contain <title ref="LIT2361somade"/> and 
+                                <title ref="LIT0000">alleluiatic table</title>, ends suddenly with the Transit of Mary. Probably part of a 
```

Comment:
as discussed, you do not need an ID here. Soma deggwa is a different case, as it may also circulate on its own.

Reply:
So I delete the title-ref for "alleluiatic table" but I keep the information in the note. Is this, what you recommend?